doc/nrf/external_comp/bt_fast_pair.rst

```diff
@@ -180,13 +180,16 @@ Provisioning registration data onto device
 The Fast Pair standard requires provisioning the device with Model ID and Anti-Spoofing Private Key obtained during device model registration.
 In the |NCS|, the provisioning data is generated as a hexadecimal file using the :ref:`bt_fast_pair_provision_script`.
 
-When building the Fast Pair in the |NCS|, the build system automatically calls the Fast Pair provision script and includes the resulting hexadecimal file in the firmware (the :file:`merged.hex` file).
+When building the Fast Pair in the |NCS|, the build system automatically calls the Fast Pair provision script and includes the resulting hexadecimal file in the final firmware hexadecimal file that can be flashed onto the device.
+For devices with the :ref:`partition_manager` support for which the :kconfig:option:`CONFIG_PM_SINGLE_IMAGE` Kconfig option is automatically enabled by the Fast Pair build system, the ``bt_fast_pair`` partition is automatically created by the build system.
+For other devices, you must declare the ``bt_fast_pair_partition`` partition manually in the device tree.
+See :file:`samples/bluetooth/fast_pair/input_device/boards/nrf54h20dk_nrf54h20_cpuapp.overlay` for example how it can be done.
```

Comment:
```suggestion
When building Fast Pair in the |NCS|, the build system automatically calls the Fast Pair provision script.
It then includes the resulting hexadecimal file in the final firmware that can be flashed onto the device.
For devices that support :ref:`partition_manager` and have the :kconfig:option:`CONFIG_PM_SINGLE_IMAGE` Kconfig option automatically enabled by the Fast Pair build system, the system also automatically creates the ``bt_fast_pair`` partition.
For other devices, you must declare the ``bt_fast_pair_partition`` partition manually in the device tree.
To see how to do this, refer to the example in the :file:`samples/bluetooth/fast_pair/input_device/boards/nrf54h20dk_nrf54h20_cpuapp.overlay` file.

Reply:
@annwoj edited your suggestion a bit. Active voice and device tree -> devicetree